bitnami/phpmyadmin/README.md

```diff
@@ -78,6 +78,9 @@ The following table lists the configurable parameters of the phpMyAdmin chart an
 | `ingress.hosts[0].tls`       | Utilize TLS backend in ingress                                                                          | `false`                                                      |
 | `ingress.hosts[0].tlsHosts`  | Array of TLS hosts for ingress record (defaults to `ingress.hosts[0].name` if `nil`)                    | `nil`                                                        |
 | `ingress.hosts[0].tlsSecret` | TLS Secret (certificates)                                                                               | `phpmyadmin.local-tls-secret`                                |
+| `securityContext.enabled`    | Enable security context for phpMyAdmin pods                                                             | `true`                                                       |
+| `securityContext.fsGroup`    | Group ID for the phpMyAdmin filesystem                                                                  | `1001`                                                       |
+| `securityContext.runAsUser`  | User ID for the phpMyAdmin container                                                                    | `1001`                                                       |
```

Comment:
We're deprecating using `securityContext` in favor of `podSecurityContext` and `containerSecurityContext` which provides more flexibility. Check Jenkins for instance:

- https://github.com/bitnami/charts/blob/master/bitnami/jenkins/values.yaml#L116
- https://github.com/bitnami/charts/blob/master/bitnami/jenkins/templates/deployment.yaml#L33
- https://github.com/bitnami/charts/blob/master/bitnami/jenkins/values.yaml#L123
- https://github.com/bitnami/charts/blob/master/bitnami/jenkins/templates/deployment.yaml#L61

As you can see the `.enabled` is also removed so we can treat the parameter as a YAML that's mapped to the specific value.

Reply:
👍